packages/react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@transifex/react",
-  "version": "7.1.3",
+  "version": "7.1.4",
```

Comment:
Better change the version whenever we want to release new version, see this PR for 7.1.3: https://github.com/transifex/transifex-javascript/commit/2d65671f9ceb7f9c8344237b0bfc71dad7496e6b